streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -17,116 +17,109 @@
 
 package org.apache.kafka.streams.kstream;
 
+import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.common.serialization.Serde;
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.processor.ProcessorSupplier;
 
 /**
- * KStream is an abstraction of an event stream in key-value pairs.
+ * KStream is an abstraction of a <i>record stream</i> in key-value pairs.
  *
  * @param <K> Type of keys
  * @param <V> Type of values
  */
+@InterfaceStability.Unstable
 public interface KStream<K, V> {
 
     /**
-     * Creates a new instance of KStream consists of all elements of this stream which satisfy a predicate
+     * Create a new instance of {@link KStream} consists of all elements of this stream which satisfy a predicate.
```

Comment:
> `Creates` vs. `Create`

Good point.  I noticed this myself earlier (and subsequently used the s-suffixed form in one of my comments by mistake I think).  My assumption for this PR has been that Kafka prefers the "Create" variant.  That said, in general I am more familiar with the "Creates" form as described in the javadoc style guide @ijuma linked above.

IMHO whatever we have done in Kafka specifically is fair game.  If in doubt though I'd favor the `Creates` form.